docker-compose.yml.erb

```diff
@@ -26,7 +27,7 @@ services:
       SENDMAIL_RELAY_PORT: 1025
       XDEBUG_REMOTE_ENABLED: 'false'
       TIDEWAYS_ENABLED: 'true'
-      RUN_BUILD: # get from host env
+      RUN_BUILD: 'false'
       WEB_DIRECTORY: public
       AUTH_HTTP_HEALTHCHECK_LOCATION: '/status/healthcheck'
     healthcheck:
```

Comment:
Just a minor thing, but since the config keys seem to be alphabetical, shouldn't `healthcheck` come after `expose`?

Reply:
Not sure that it was intentionally so (we tend to keep image at the top, for example)

Reply:
it was put at the bottom at the time because it was very related to the next healthcheck: key